apps/frontend/app/sitemap.ts

```diff
@@ -0,0 +1,18 @@
+import { envUtil } from 'src/utils';
+
+import type { MetadataRoute } from 'next';
+
+const baseUrl = envUtil.getEnv().frontendUrl;
+
+export default function sitemap(): MetadataRoute.Sitemap {
+  return [
+    {
+      url: `${baseUrl}/`,
```

Comment:
Declare list of URLs above and then map it to the sitemap objects to reduce code duplication